cmd/main.go

```diff
@@ -2,29 +2,18 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
-	"log"
-	"net/http"
 	"os"
+	"weather"
 )
 
 func main() {
 
 	token := os.Getenv("WEATHERAPP_TOKEN")
 	if len(token) == 0 {
-		fmt.Printf("please set a weatherapp token\n")
+		fmt.Fprintf(os.Stderr, "please set env variable, WEATHERAPP_TOKEN \n")
```

Comment:
It seems like getting the token from the environment is something that `CliOutput` could do, doesn't it? After all, that handles the command-line parsing. Environment variables are just another way of supplying information to the program.